src/StatusQ/Controls/Validators/StatusMinLengthValidator.qml

```diff
@@ -6,6 +6,12 @@ StatusValidator {
 
     name: "minLength"
 
+    errorMessage: {
+        minLength === 1 ?
+            "Please enter a value" :
+            `Value is ${errors.address.actual} characters, but must be at least ${minLength}.`
```

Comment:
Just realised that this won't work as we don't have access to `errors` here. And it makes sense, because only the input knows about all of its validators and which ones have failed. Probably the easiest to get "around" it is to just remove the `actual` value from the default message.

Will think about it.